src/admin-views/editor/fieldset/advanced.php

```diff
@@ -52,6 +52,24 @@
 			class="ticket_field ticket_form_right"
 			id="ticket_description"
 		><?php echo esc_textarea( $ticket ? $ticket->description : null ) ?></textarea>
+		<div class="input_block">
+			<label class="tribe_soft_note">
+				<input
+						type="checkbox"
+						id="tribe_tickets_show_type"
+						name="ticket_show_type"
+						value="1"
+						class="ticket_field ticket_form_left"
+						<?php checked( true, $ticket ? $ticket->show_type : true ); ?>
+				>
+				<?php
+				echo esc_html( sprintf(
+						__( 'Show type on front end %s form.', 'event-tickets' ),
+						tribe_get_ticket_label_singular_lowercase( 'default_ticket_provider' )
```

Comment:
:no_entry_sign: Multi-line function call not indented correctly; expected 20 spaces but found 24

<sub>hash: f0b98b8b78e7c1059c49f344c4db5350</sub>